<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceCMDCtrlDlg</class>
 <widget class="QDialog" name="DeviceCMDCtrlDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>780</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>设备反控</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(186, 189, 182);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,2,2,3,1,5">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="3,12,0,1">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>设备指令控制</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="timertk">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="test_sendintime">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	background-color: rgb(100, 85, 0);
	border-radius:5px;
	color: rgb(255, 255, 255);
	font: 75 16pt &quot;微软雅黑&quot;;
}

QPushButton::hover
{
	
	
	background-color: rgb(110, 65, 0);
}
</string>
       </property>
       <property name="text">
        <string>测试定时发送</string>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_quit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	background-color: rgb(100, 85, 0);
	border-radius:5px;
	color: rgb(255, 255, 255);
	font: 75 16pt &quot;微软雅黑&quot;;
}

QPushButton::hover
{
	
	
	background-color: rgb(110, 65, 0);
}
</string>
       </property>
       <property name="text">
        <string>退出</string>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0,1">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 75 18pt &quot;微软雅黑&quot;;
</string>
       </property>
       <property name="text">
        <string>选择设备因子</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="curDevice">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox{
  color:#000000;
  font: 75 14pt &quot;微软雅黑&quot;;
  padding: 1px 15px 1px 3px;
  border:1px solid rgba(228,228,228,1);
  border-radius:5px 5px 0px 0px;
	background-color: rgb(255, 255, 255);
} 

  QComboBox::drop-down {
      subcontrol-origin: padding;
      subcontrol-position: top right;
      width: 15px;
      border:none;
  }

  QComboBox::down-arrow {
      
	image: url(:/new/images/image/down_arrow.png);
  }

  QComboBox QAbstractItemView{
	background:rgba(255,255,255,1);
    border:1px solid rgba(228,228,228,1);
    border-radius:0px 0px 5px 5px;
	font: 75 18pt &quot;微软雅黑&quot;;
    outline: 0px;  
  }

QComboBox QAbstractItemView::item{
	height:36px;
	color:#666666;
	padding-left:9px;
	background-color:#FFFFFF;
	font: 75 14pt &quot;微软雅黑&quot;;
}
QComboBox QAbstractItemView::item:hover{ 
  background-color:#409CE1;
  color:#ffffff;
}
QComboBox QAbstractItemView::item:selected{
  background-color:#409CE1;
  color:#ffffff;
}</string>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtTop</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 75 18pt &quot;微软雅黑&quot;;
</string>
       </property>
       <property name="text">
        <string>选择串口</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="curPort">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox{
  color:#000000;
  font: 75 14pt &quot;微软雅黑&quot;;
  padding: 1px 15px 1px 3px;
  border:1px solid rgba(228,228,228,1);
  border-radius:5px 5px 0px 0px;
	background-color: rgb(255, 255, 255);
} 

  QComboBox::drop-down {
      subcontrol-origin: padding;
      subcontrol-position: top right;
      width: 15px;
      border:none;
  }

  QComboBox::down-arrow {
      
	image: url(:/new/images/image/down_arrow.png);
  }

  QComboBox QAbstractItemView{
	background:rgba(255,255,255,1);
    border:1px solid rgba(228,228,228,1);
    border-radius:0px 0px 5px 5px;
	font: 75 18pt &quot;微软雅黑&quot;;
    outline: 0px;  
  }

QComboBox QAbstractItemView::item{
	height:36px;
	color:#666666;
	padding-left:9px;
	background-color:#FFFFFF;
	font: 75 14pt &quot;微软雅黑&quot;;
}
QComboBox QAbstractItemView::item:hover{ 
  background-color:#409CE1;
  color:#ffffff;
}
QComboBox QAbstractItemView::item:selected{
  background-color:#409CE1;
  color:#ffffff;
}</string>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtTop</enum>
       </property>
       <item>
        <property name="text">
         <string>COM1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>USB-COM1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>USB-COM2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>USB-COM3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>USB-COM4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>XR-COM1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>XR-COM2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>XR-COM3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>XR-COM4</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="cominfo">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>无连接</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,2,2,1,1,1">
     <item>
      <widget class="QPushButton" name="btnJZ">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	
	
	background-color: rgb(0, 85, 0);
	border-radius:5px;
	color: rgb(255, 255, 255);
	font: 75 14pt &quot;微软雅黑&quot;;
}

QPushButton::hover
{
	
	
	background-color: rgb(0, 170, 0);
}
</string>
       </property>
       <property name="text">
        <string>零点校准/量程校准</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btntestnow">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	
	
	background-color: rgb(0, 85, 0);
	border-radius:5px;
	color: rgb(255, 255, 255);
	font: 75 14pt &quot;微软雅黑&quot;;
}

QPushButton::hover
{
	
	
	background-color: rgb(0, 170, 0);
}
</string>
       </property>
       <property name="text">
        <string>即时采样/立即测量</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnfc">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	
	
	background-color: rgb(0, 85, 0);
	border-radius:5px;
	color: rgb(255, 255, 255);
	font: 75 14pt &quot;微软雅黑&quot;;
}

QPushButton::hover
{
	
	
	background-color: rgb(0, 170, 0);
}
</string>
       </property>
       <property name="text">
        <string>启动清洗/反吹</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBDCY">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	
	
	background-color: rgb(0, 85, 0);
	border-radius:5px;
	color: rgb(255, 255, 255);
	font: 75 14pt &quot;微软雅黑&quot;;
}

QPushButton::hover
{
	
	
	background-color: rgb(0, 170, 0);
}
</string>
       </property>
       <property name="text">
        <string>比对采样</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteSend">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	
	
	
	background-color: rgb(245, 121, 0);
	border-radius:5px;
	color: rgb(255, 255, 255);
	font: 75 14pt &quot;微软雅黑&quot;;
}

QPushButton::hover
{
	
	
	
	background-color: rgb(206, 92, 0);
}
</string>
       </property>
       <property name="text">
        <string>发送清除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteReceive">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	
	
	
	background-color: rgb(52, 101, 164);
	border-radius:5px;
	color: rgb(255, 255, 255);
	font: 75 14pt &quot;微软雅黑&quot;;
}

QPushButton::hover
{
	
	
	
	background-color: rgb(32, 74, 135);
}
</string>
       </property>
       <property name="text">
        <string>接收清除</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="timeset"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,6,1,1">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 18pt &quot;微软雅黑&quot;;
</string>
       </property>
       <property name="text">
        <string>发送内容</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sendEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_sendcmd">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	
	background-color: rgb(0, 85, 255);
	border-radius:5px;
	color: rgb(255, 255, 255);
	font: 75 22pt &quot;微软雅黑&quot;;
}

QPushButton::hover
{
	
	background-color: rgb(0, 129, 194);
}
</string>
       </property>
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_sendbytime">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>18</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	
	background-color: rgb(30, 120, 0);
	border-radius:5px;
color: rgb(255, 255, 255);
	font: 75 18pt '微软雅黑';
}

QPushButton::hover
{
	
	background-color: rgb(129, 194, 0);
}
</string>
       </property>
       <property name="text">
        <string>启动定时发送</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,5,1">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 18pt &quot;微软雅黑&quot;;
</string>
       </property>
       <property name="text">
        <string>接收内容</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="receiveEdit">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="plainText">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="keyboard">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>10</y>
            <width>41</width>
            <height>41</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/images/keyboard-off.png</normaloff>:/images/keyboard-off.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QCheckBox" name="loopread">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>91</width>
            <height>29</height>
           </rect>
          </property>
          <property name="text">
           <string>循环读取</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
